Core/Core/Database/CoreData/Models/SubmissionComment.swift

```diff
@@ -30,12 +30,18 @@ final public class SubmissionComment: NSManagedObject {
     @NSManaged public var mediaTypeRaw: String?
     @NSManaged public var mediaURL: URL?
     @NSManaged public var submissionID: String
+    @NSManaged public var fileIDsRaw: String?
```

Comment:
Assignment has `@NSManaged public var allowedExtensions: [String]` Can we not use `[String]?`? Or do we need to differentiate between empty and nil?

Reply:
I'm going to use a relationship (later).